torchvision/io/__init__.py

```diff
@@ -20,10 +23,75 @@
     encode_jpeg,
     write_jpeg,
     encode_png,
-    write_png
+    write_png,
 )
 
 
+if _HAS_VIDEO_OPT:
+
+    class Video:
```

Comment:
For a follow-up PR: given that we are wrapping our C++ class in a python class, we don't need to have this as is, and we can instead implement everything outside the guard, but have a guard at instantiation time.

Something like
```python
if _HAS_VIDEO_OPT:
    def _has_video_opt():
        return True
else:
    def _has_video_opt():
        return False

class Video:
    def __init__(self, path, stream):
        # check if video_opt is available
        if not _has_video_opt():
            raise RuntimeError(...)
        ...
```
In here, we create a function `_has_video_opt` to make the class work with torchscript (which doesn't support globals).

Reply:
added to the list of issues to address